q2_feature_classifier/_cutter.py

```diff
@@ -168,6 +168,10 @@ def extract_reads(sequences: DNASequencesDirectoryFormat, f_primer: str,
     q2_types.DNAFASTAFormat
         containing the reads
     """
+    if min_length > trunc_len and trunc_len > 0:
+        raise ValueError('Your minimum length is larger than your truncation '
+                         'length, this will result in all of your sequences '
+                         'being trimmed automatically.')
```

Comment:
"trimmed automatically." --> "removed from your dataset. To proceed, you must set a minimum length less than your truncation length."